Makefile

```
@@ -49,18 +49,21 @@ help: ## Show this help message
 # ==============================================================================
 
 .PHONY: setup
-setup: ## Initialize project: install dependencies, create .env files and pull required Docker images.
+setup: ## Initialize project: install dependencies, create .env file and pull required Docker images.
```

checkmake (0.2.2)

[warning] Target body for "setup" exceeds allowed length of 5 (16).

(maxbodylength)